src/screens/signup/index.tsx

```diff
@@ -27,64 +28,71 @@ export default function SignupScreen(props: SignupScreenProp) {
   const [email, setEmail] = useState('');
   return (
     <Container>
-        <ScrollView showsVerticalScrollIndicator={false}>
-      <SafeAreaView>
-        <Welcome>Welcome</Welcome>
-        <Prompt>SIGN UP OR LOG IN</Prompt>
-        <Input
-          inputLabel="Email address"
-          placeholder="tashacombs@ymmail.com"
-          defaultValue={email}
-          onChangeText={input => setEmail(input)}
-          inputLabelStyle={{ fontSize: fonts.LARGE_SIZE + 3 }}
-          textInputStyle={{
-            height: 60,
-            borderColor: colors.FONT_DARK_COLOR,
-            borderBottomWidth: 1,
-            fontSize: fonts.LARGE_SIZE
-          }}
-        />
-        <Button
-          title="Get Started with email"
-          buttonStyle={{ marginTop: '6%', height: 60 }}
-          onPress={() => props.navigation.navigate('createAccountScreen')}
-        />
-        <Choice>or</Choice>
-        <Button
-          title="Continue with facebook"
-          buttonStyle={{
-            marginBottom: '6%',
-            height: 60,
-            borderColor: colors.FACEBOOK_COLOR,
-            borderWidth: 1,
-            backgroundColor: colors.BG_WHITE_COLOR
-          }}
-          textStyle={{ color: colors.FACEBOOK_COLOR }}
-        >
-          <IconView>
-            <Ionicons name="logo-facebook" size={30} color="#4267B2" />
-          </IconView>
-        </Button>
-        <Button
-          title="Continue with Google"
-          buttonStyle={{
-            marginBottom: '6%',
-            height: 60,
-            borderColor: colors.GOOGLE_COLOR,
-            borderWidth: 1,
-            backgroundColor: colors.BG_WHITE_COLOR
-          }}
-          textStyle={{ color: colors.GOOGLE_COLOR }}
-        >
-          <IconView>
-            <GoogleIcon source={require('../../../assets/icons/google.png')} />
-          </IconView>
-        </Button>
-        <Terms>By signing up, you agree to our 
-          <Service> Terms of Service</Service>
-        </Terms>
-      </SafeAreaView>  
+      <ScrollView showsVerticalScrollIndicator={false}>
+        <SafeAreaView>
```

Comment:
SafeAreaView before ScrollView, SafeAreaView always first before anything on the page